docs/src/inverse_problems/global_sensitivity_analysis.md

```diff
@@ -142,7 +142,7 @@ Here, the function's sensitivity is evaluated with respect to each output indepe
 
 ---
 ## [Citations](@id global_sensitivity_analysis_citations)
-If you use this functionality in your research, [in addition to Catalyst](@ref catalyst_citation), please cite the following paper to support the authors of the GlobalSensitivity.jl package:
+If you use this functionality in your research, [in addition to Catalyst](@ref doc_index_citation), please cite the following paper to support the authors of the GlobalSensitivity.jl package:
```

Comment:
I changed the crosss reference tags to all start with the tag for the doc page, so had to change the places where we referenced the citation place.